src/databricks/labs/ucx/hive_metastore/table_size.py

```diff
@@ -23,20 +22,25 @@ class TableSize:
 
 
 class TableSizeCrawler(CrawlerBase[TableSize]):
-    def __init__(self, backend: SqlBackend, schema, include_databases: list[str] | None = None):
+    # TODO: Ensure TablesCrawler and FasterTableScanCrawler share a common interface.
```

Comment:
@pritishpai : Maybe could be part of: https://github.com/databrickslabs/ucx/pull/2819